test/routes/routes.spec.unit.ts

```diff
@@ -1,8 +1,18 @@
 import * as request from "supertest";
 import app from "../../src/app";
+import { COOKIE_NAME } from "../../src/properties";
 
 describe("Basic URL Tests", () => {
 
+  it("should find start page", async () => {
+    const response = await request(app)
+        .get("/strike-off-objections")
+        .set("Cookie", [`${COOKIE_NAME}=123`]);
```

Comment:
what happens if the user is new and doesn't have a cookie? we should test that as well.

Reply:
As it is the landing page the result should be the same. Will add a test to show this.